### ISSUE TITLE
Create doc list with Ya.Ecommerce and Ya.Aims features

### ISSUE THREAD

**Issue by duker33** [fidals/shopelectro#473]
Ancestor of #292 

Write just list of already supported and for coming soon.

Possible structure:
```
# Яндекс.Екоммерс
## Поддерживается
- Фича 1
- Фича 2
## В разработке
- Фича 3
- Фича 4

# Яндекс.Цели
## Поддерживается
- Фича 1
- Фича 2
## В разработке
- Фича 3
- Фича 4
```

**Comment by 0pdd** [comment]
@duker33 the puzzle [#489](https://github.com/fidals/shopelectro/issues/489) is still not solved.

**Comment by 0pdd** [comment]
@duker33 3 puzzles [#489](https://github.com/fidals/shopelectro/issues/489), [#493](https://github.com/fidals/shopelectro/issues/493), [#494](https://github.com/fidals/shopelectro/issues/494) are still not solved.

**Comment by 0pdd** [comment]
@duker33 6 puzzles [#489](https://github.com/fidals/shopelectro/issues/489), [#493](https://github.com/fidals/shopelectro/issues/493), [#494](https://github.com/fidals/shopelectro/issues/494), [#496](https://github.com/fidals/shopelectro/issues/496), [#497](https://github.com/fidals/shopelectro/issues/497), [#498](https://github.com/fidals/shopelectro/issues/498) are still not solved.

**Comment by 0pdd** [comment]
@duker33 6 puzzles [#489](https://github.com/fidals/shopelectro/issues/489), [#494](https://github.com/fidals/shopelectro/issues/494), [#496](https://github.com/fidals/shopelectro/issues/496), [#497](https://github.com/fidals/shopelectro/issues/497), [#498](https://github.com/fidals/shopelectro/issues/498), [#515](https://github.com/fidals/shopelectro/issues/515) are still not solved; solved: [#493](https://github.com/fidals/shopelectro/issues/493).

**Comment by 0pdd** [comment]
@duker33 5 puzzles [#489](https://github.com/fidals/shopelectro/issues/489), [#494](https://github.com/fidals/shopelectro/issues/494), [#497](https://github.com/fidals/shopelectro/issues/497), [#498](https://github.com/fidals/shopelectro/issues/498), [#515](https://github.com/fidals/shopelectro/issues/515) are still not solved; solved: [#493](https://github.com/fidals/shopelectro/issues/493), [#496](https://github.com/fidals/shopelectro/issues/496).

**Comment by 0pdd** [comment]
@duker33 4 puzzles [#489](https://github.com/fidals/shopelectro/issues/489), [#494](https://github.com/fidals/shopelectro/issues/494), [#497](https://github.com/fidals/shopelectro/issues/497), [#498](https://github.com/fidals/shopelectro/issues/498) are still not solved; solved: [#493](https://github.com/fidals/shopelectro/issues/493), [#496](https://github.com/fidals/shopelectro/issues/496), [#515](https://github.com/fidals/shopelectro/issues/515).

**Comment by 0pdd** [comment]
@duker33 4 puzzles [#489](https://github.com/fidals/shopelectro/issues/489), [#497](https://github.com/fidals/shopelectro/issues/497), [#498](https://github.com/fidals/shopelectro/issues/498), [#696](https://github.com/fidals/shopelectro/issues/696) are still not solved; solved: [#493](https://github.com/fidals/shopelectro/issues/493), [#494](https://github.com/fidals/shopelectro/issues/494), [#496](https://github.com/fidals/shopelectro/issues/496), [#515](https://github.com/fidals/shopelectro/issues/515).

**Comment by 0pdd** [comment]
@duker33 2 puzzles [#489](https://github.com/fidals/shopelectro/issues/489), [#696](https://github.com/fidals/shopelectro/issues/696) are still not solved; solved: [#493](https://github.com/fidals/shopelectro/issues/493), [#494](https://github.com/fidals/shopelectro/issues/494), [#496](https://github.com/fidals/shopelectro/issues/496), [#497](https://github.com/fidals/shopelectro/issues/497), [#498](https://github.com/fidals/shopelectro/issues/498), [#515](https://github.com/fidals/shopelectro/issues/515).

**Comment by 0pdd** [comment]
@duker33 the puzzle [#489](https://github.com/fidals/shopelectro/issues/489) is still not solved; solved: [#493](https://github.com/fidals/shopelectro/issues/493), [#494](https://github.com/fidals/shopelectro/issues/494), [#496](https://github.com/fidals/shopelectro/issues/496), [#497](https://github.com/fidals/shopelectro/issues/497), [#498](https://github.com/fidals/shopelectro/issues/498), [#515](https://github.com/fidals/shopelectro/issues/515), [#696](https://github.com/fidals/shopelectro/issues/696).
